inference-engine/src/mkldnn_plugin/emitters/jit_eltwise_emitters.cpp

```diff
@@ -1224,16 +1287,49 @@ size_t jit_logical_not_emitter::aux_vecs_count() const {
 }
 
 /// POWER_STATIC ///
+jit_power_static_emitter::jit_power_static_emitter(jit_generator *host, cpu_isa_t host_isa, const std::shared_ptr<ngraph::Node>& node, Precision exec_prc)
+: jit_emitter(host, host_isa, node, exec_prc) {
+    auto parent = node->input(1).get_source_output().get_node_shared_ptr();
+    if (!std::dynamic_pointer_cast<ngraph::opset1::Constant>(parent)) {
+        throw ngraph::ngraph_error("unsupported non constant power");
+    }
+
+    if (!(node->input(1).get_shape() == ngraph::Shape() || ngraph::shape_size(node->input(1).get_shape()) == 1)) {
+        throw ngraph::ngraph_error("unsupported non scalar power");
+    }
+    power = ngraph::as_type_ptr<ngraph::op::Constant>(parent)->get_data_ptr<float>()[0];
+    scale = 1.f;
+    shift = 0.f;
+    push_arg_entry_of("power", float2int(power), true);
+    push_arg_entry_of("scale", 0x3f800000, true);
+    push_arg_entry_of("shift", 0x00000000, true);
+    push_arg_entry_of("one",   0x3f800000, true);
+
+    prepare_table();
+}
 jit_power_static_emitter::jit_power_static_emitter(jit_generator *host, cpu_isa_t host_isa, const MKLDNNNode* node, Precision exec_prc)
 : jit_emitter(host, host_isa, node, exec_prc) {
+    auto *powerLayer = dynamic_cast<InferenceEngine::PowerLayer *>(node->getCnnLayer().get());
+    if (powerLayer == nullptr)
+        THROW_IE_EXCEPTION << "Cannot convert power layer.";
+
+    power = powerLayer->power;
+    scale = powerLayer->scale;
+    shift = powerLayer->offset;
+
+    push_arg_entry_of("power", float2int(power), true);
```

Comment:
As soon as you preserve values of power, scale and shift I think `push_arg_entry_of` should still be in `register_table_entries` since it is ideologically right place.